react-toggle-switch/my-app/src/App.tsx

```diff
@@ -0,0 +1,33 @@
+import { useState } from 'react';
+import { ToggleSwitch } from './ToggleSwitch.tsx';
+import './App.css';
+
+function App() {
+  const [switchStatus, setSwitchStatus] = useState(true);
+
+  let labelText = '';
+  if (switchStatus) {
+    labelText = 'On';
+  } else {
+    labelText = 'Off';
+  }
+
+  const switchClass = labelText.toLowerCase();
+
+  function handleToggle() {
+    if (switchStatus) {
```

Comment:
More simply, you can just use `!` to toggle the value: `setSwitchStatus(!switchStatus)`